svc/IdleService.js

```diff
@@ -66,6 +66,7 @@ export class IdleService {
         if (XH.appState != AppState.SUSPENDED) {
             XH.setAppState(AppState.SUSPENDED);
             this.destroyAppListeners();
+            XH.webSocketService.shutdown();
```

Comment:
Great, agree better to have this handled here so we can look in one place to understand what suspending does.